.github/workflows/cla.yml

```diff
@@ -0,0 +1,29 @@
+name: "CLA Assistant"
+
+on:
+  issue_comment:
+    types:
+      - "created"
+  pull_request_target:
+    types:
+      - "opened"
+      - "closed"
+      - "synchronize"
+
+jobs:
+  cla-assistant:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "CLA Assistant"
+        if: "(github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'"
+        uses: "cla-assistant/github-action@v2.1.3-beta"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          PERSONAL_ACCESS_TOKEN: "${{ secrets.CLA_ASSISTANT_ACCESS_TOKEN }}"
+        with:
+          remote-organization-name: "localstack"
+          remote-repository-name: "localstack"
+          path-to-signatures: "v1/signatures.json"
+          path-to-document: "https://github.com/localstack/localstack/blob/master/.github/CLA.md"
+          branch: "master"
```

Comment:
We should either change the branch to `cla-signatures` here, or adjust our CI config to skip running tests for commits to this file. I'm more leaning towards the former (to keep the build config clean), but open to discussion. Alternatively, we could also push the signatures to a private repo over time..